entry/.preview/intermediates/res/debug/rich/assets/js/default/pages/index/index.js

```diff
@@ -0,0 +1,1185 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
Please remove .preview folder.... If any more such hidden folder or intermediate folders to be removed